io.vikunja.Desktop.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.vikunja.Desktop
+app-version: '0.21.0'
+version-hash: '46bd28e13d35f78bbcf2d3b44307059b195f5d1eefcd08937645888f008365e3'
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+command: vikunja-desktop
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
do you really need this?

Reply:
All of them?
Notifications yeah

Reply:
line 23 is the subject.

Reply:
@RaptaG what do you mean?

Reply:
Doesn't Vikunja notify you when a tasks time is up?

Reply:
Ah it's via e-mail nvm